app/src/ui/components/Card.module.css

```diff
@@ -6,6 +6,14 @@
   border: 1px solid;
 
   padding: 50px 43px;
+
+  @media (max-width: 430px) {
+    width: 65%;
+  }
+
+  @media (max-width: 768px) {
+    transform: scale(0.85);
+  }
```

Comment:
what if you have this card in other places? probably it would require another styling for mobile versions depending on context

so I suggest to move this styles to `App.css` and apply it to the `Card` component via `className`